library/core/src/iter/traits/iterator.rs

```diff
@@ -1990,6 +1990,44 @@ pub trait Iterator {
         accum
     }
 
+    /// An iterator method that applies a function, producing a single, final value.
+    ///
+    /// `fold_mut()` is very similar to [`fold()`] except that the closure
+    /// takes a `&mut` to the 'accumulator' and does not need to return a new value.
+    ///
+    /// [`fold()`]: Iterator::fold
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// #![feature(iterator_fold_mut)]
+    /// use std::collections::HashMap;
+    ///
+    /// let word = "abracadabra";
+    ///
+    /// // the count of each letter in a HashMap
+    /// let counts = word.chars().fold_mut(HashMap::new(), |map, c| {
+    ///   *map.entry(c).or_insert(0) += 1;
+    /// });
+    ///
+    /// assert_eq!(counts[&'a'], 5);
+    /// ```
+    #[inline]
+    #[unstable(feature = "iterator_fold_mut", issue = "76725")]
```

Comment:
This `issue` needs to be a real tracking issue, not the feature request.
See other instances of [issues labeled `C-tracking-issue` and `T-libs`](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AC-tracking-issue+label%3AT-libs).

Reply:
Hmm I created https://github.com/rust-lang/rust/issues/76751 but I don't obviously see how to add T-libs to it 🤔

Reply:
It got tagged but in the future you can use @rustbot modify labels: +(label), -(label)

Reply:
d7cc6bec0e765ec3fb03169a15ddca9efbc6bdc9